src/NServiceBus.RavenDB/Outbox/OutboxPersister.cs

```diff
@@ -1,14 +1,18 @@
 ﻿namespace NServiceBus.RavenDB.Outbox
 {
     using System;
+    using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
     using NServiceBus.Extensibility;
     using NServiceBus.Outbox;
     using Raven.Client;
+    using TransportOperation = NServiceBus.Outbox.TransportOperation;
 
     class OutboxPersister : IOutboxStorage
     {
+        public string EndpointName { get; set; }
+
         public OutboxPersister(IDocumentStore documentStore)
```

Comment:
Should OutboxPersister require EndpointName as a constructor argument? @DavidBoike